arrays/arrays2D/Solution1Arrays.go

```diff
@@ -0,0 +1,23 @@
+package Arrays2D
+
+import "fmt"
+
+type ImplS1A struct{}
+
+func (s1 ImplS1A) arrays2D() string {
+	arr := [3][4]int{
+		{1, 2, 3, 4},
+		{2, 4, 6, 8},
+		{1, 3, 5, 7},
+	}
+
+	var out string
+
+	for i := 0; i < 3; i++ {
```

Comment:
never hard code the values; use the len func here for both dimensions; its little tricky for the first time, do let me know if you need some direction here

Reply:
fixed